Sming/Core/Network/Http/HttpClientConnection.cpp

```diff
@@ -315,6 +315,7 @@ void HttpClientConnection::onClosed()
 		System.queueCallback([this]() {
 			cleanup();
 			init(HTTP_RESPONSE);
+			state = eHCS_Ready;
```

Comment:
Should this be done in `init()` ?

Reply:
> Should this be done in init() ?

Yes, indeed. I will add this correction in a second...